### PR TITLE
Update PetPet.lua

### PR DESCRIPTION
**Pull request by Toodo**
this will fix a problem when u land and collecting the gas cloud. before this u will summon your pet in middle of the channeling its about this tool https://www.wowhead.com/wotlk/item=23821/zapthrottle-mote-extractor

### CODE REVIEW

**Comment by neurotech** [comment]
Thanks @Toodo ! I am currently unsubscribed so I haven't been able to test any code changes I've made.

Can I please ask that you include the fix you mentioned on #2 on this PR as well?

File: Elements/Elements_Checkboxes.lua [hunk 1]
```diff
@@ -41,9 +41,9 @@ Elements.Checkboxes.Create = function(labelText, parent, anchorPoint, relativePo
   checkbox:SetBackdropBorderColor(unpack(Elements.Palette.RGB.BLACK))
   checkbox:SetCheckedTexture(Elements.Textures.BUTTON_BG_TEXTURE)
   checkbox:SetNormalTexture(Elements.Textures.BUTTON_BG_TEXTURE)
-  checkbox:SetHighlightTexture(nil)
-  checkbox:SetPushedTexture(nil)
-  checkbox:SetDisabledTexture(nil)
+--  checkbox:SetHighlightTexture(nil)
+--  checkbox:SetPushedTexture(nil)
+--  checkbox:SetDisabledTexture(nil)
```
Comment (neurotech) [inline Elements/Elements_Checkboxes.lua]: Might as well just remove these comments at this point:

```suggestion
```

**Review by neurotech — CHANGES_REQUESTED** [review]

File: Elements/Elements_Checkboxes.lua (hunk 1 above)
Comment (Toodo) [reply Elements/Elements_Checkboxes.lua]: yes sounds good, it works without them so

**Review by neurotech — APPROVED** [review]

**Comment by neurotech** [comment]
LGTM @Toodo. Thanks again!

**Comment by neurotech** [comment]
Resolves #2

**Pull request merged**